docs/source/raft_configuration.md

```diff
@@ -317,6 +317,12 @@ monitor:
    they can be shared with the consenter set. If this value begins to climb, this
    node may not be able to participate in consensus (which could lead to a
    service interruption for this node and possibly the network).
+* `consensus_etcdraft_cluster_size` and `consensus_etcdraft_active_nodes`: these
+   channel metrics help track the "active" nodes (which, as it sounds, are the nodes that
+   are currently contributing to the cluster, as compared to the total number of
+   nodes in the cluster). If the number of active nodes falls below a majority of
```

Comment:
Not sure if it would be redundant or useful to clarify that "the ordering service will stop processing blocks for the channel". Just quickly reading this makes it sound like the orderer will stop processing _all_ blocks but I'll defer to you based on what would be consistent with our other docs for the orderer/raft/consensus.

Reply:
My assumption was that people would understand that "stop processing blocks" referred to just the channel in question (since it's a channel metric), though unless channels have different sets of consenters it likely would be true for at least several channels. But I'm happy to add a slight clarification here to say "stop processing blocks on the channel".